TestFoundation/TestNSSet.swift

```diff
@@ -237,9 +237,14 @@ class TestNSSet : XCTestCase {
         let set: NSSet = NSSet(array: [array, dictionary, innerSet, 1111, 2222, 3333])
         
         let description = NSString(string: set.description)
-        
+
+#if DARWIN_COMPATIBILITY_TESTS
+        XCTAssertTrue((description.substring(to: 2) as NSString).isEqual(to: "{("))
```

Comment:
These indicate that we have an incorrect signature  or missing overload for `.substring(…)` in SCF. We should fix that instead.

Reply:
This is also https://bugs.swift.org/browse/SR-10916 which I have rewritten to use `.hasPrefix()` and `.hasSuffix()`